yktrace/src/hwt/mapper.rs

```diff
@@ -31,6 +32,10 @@ impl HWTMapper {
     }
 
     /// Maps each entry of a hardware trace to the appropriate SirLoc.
+    ///
+    /// For each block in the trace, the interval tree is queried for labels coinciding with the
+    /// block. Label addresses which coincide are therefore contained within the block, and are
+    /// thus part of the SIR trace.
     pub fn map(&self, trace: Box<dyn Trace>) -> Result<Vec<SirLoc>, HWTracerError> {
```

Comment:
I wonder if we can use a slightly more specific name than `map`? Just because `map` has a slightly generic meaning in Rust...

Reply:
Fixed in 0df753f

Reply:
Works for me.